code/modules/paperwork/pen.dm

```diff
@@ -65,7 +65,7 @@
 	if(..())
 		if(reagents.total_volume)
 			if(M.reagents)
-				reagents.trans_to(M, 50)
+				reagents.trans_to(M, 55)

```

Comment:
I believe this can be future-proofed by changing 55 to reagents.maxiumum_volume. Test to be sure though, because I aint too familiar with reagents code.